examples/resources/opslevel_property_assignment/resource.tf

```diff
@@ -0,0 +1,17 @@
+resource "opslevel_property_assignment" "example" {
+  definition = "Z2lkOi8vb3BzbGV2ZWwvUHJvcGVydGllczo6RGVmaW5pdGlvbi8xODA"
+  owner = "Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS85MTQyOQ"
+  value = jsonencode("green")
+}
+
+resource "opslevel_property_assignment" "example_2" {
+  definition = "Z2lkOi8vb3BzbGV2ZWwvUHJvcGVydGllczo6RGVmaW5pdGlvbi85OA"
+  owner = "Z2lkOi8vb3BzbGV2ZWwvU2VydmljZS85MTQyOQ"
+  value = "true"
```

Comment:
It's equivalent, but what do you think about `jsonencode(true)` here?

Reply:
Great suggestion - mixing `jsonencode` and strings in the examples is confusing, so I replaced this with `jsonencode(true)`.